### PR TITLE
Added endpoint to add schedules

### PR DESCRIPTION
**Pull request by santostiago**
Added a new endpoint to add the schedules.
The expected payload is the following:

```
{
	"store-id": 5,
	"timetable":
	{
		"sunday": {
			"o": "10:30",
			"c": "15:20"
		},
		"monday": {
			"o": "12:00",
			"c": "15:00"
		},
		"tuesday": {
			"o": "12:00",
			"c": "15:00"
		},
		"wednesday": {
			"o": "12:00",
			"c": "15:00"
		},
		"thursday": {
			"o": "12:00",
			"c": "15:00"
		},
		"friday": {
			"o": "12:00",
			"c": "15:00"
		},
		"saturday": {
			"o": "12:00",
			"c": "15:00"
		}	
	}
}
```

### CODE REVIEW

File: app/controllers/api/v1/schedules_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+module Api
+  module V1
+    class SchedulesController < ExternalApiController
+      # This Endpoint is not using JSON API format!
+      def create
+        return forbidden unless current_user.admin?
+
+        store = Store.find_by(id: params['store-id'])
```
Comment (simaob) [inline app/controllers/api/v1/schedules_controller.rb:8]: Why not `Store.find`? =)

File: app/controllers/api/v1/schedules_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        end
+      end
+
+      private
+
+      def add_timetable(store, timetable)
+        errors = []
+        WeekDay.transaction do
+          WeekDay.days.keys.each do |day|
+            day_time = WeekDay.find_or_create_by(store_id: store.id, day: day)
```
Comment (simaob) [inline app/controllers/api/v1/schedules_controller.rb:28]: Could use `find_or_initialize_by`, and only save after the data is added.

File: app/controllers/api/v1/schedules_controller.rb (hunk 1 above)
Comment (santostiago) [reply app/controllers/api/v1/schedules_controller.rb:8]: @simaob 
AR's `.find` throws an exception when it cannot find anything.

File: app/controllers/api/v1/schedules_controller.rb (hunk 2 above)
Comment (santostiago) [reply app/controllers/api/v1/schedules_controller.rb:28]: @simaob true. I'm going to change it.

**Pull request merged**